pkg/asset/quota/quota.go

```diff
@@ -118,7 +120,17 @@ func (a *PlatformQuotaCheck) Generate(dependencies asset.Parents) error {
 			return summarizeFailingReport(reports)
 		}
 		summarizeReport(reports)
-	case azure.Name, baremetal.Name, libvirt.Name, none.Name, openstack.Name, ovirt.Name, vsphere.Name:
+	case typesopenstack.Name:
+		ci, err := openstackvalidation.GetCloudInfo(ic.Config)
+		if err != nil {
+			return errors.Wrap(err, "failed to get cloud info")
+		}
+		reports, err := quota.Check(ci.Quotas, openstack.Constraints(ic.Config, masters, workers))
```

Comment:
You can pass a reference to the CloudInfo object to `Constraints()` now, in order to not call `GetCloudInfo()` from the [pkg/asset/quota/openstack/openstack.go](https://github.com/openshift/installer/pull/4432/files#diff-db1151cdf97e6679829445032d2f0815774f20dfd956c48d2863321979f098fe) file.

Reply:
ack, I'm looking at it now.